src/test/resources/logging.properties

```diff
@@ -17,7 +17,7 @@
 #
 
 # Additional logger names to configure (root logger is always configured)
-loggers=org.apache.directory,net.sf.ehcache,org.apache.directory.api.ldap,javax.security.sasl,org.wildfly.security,org.wildfly.security.sasl,org.wildfly.security.xml,org.wildfly.security.tls
+loggers=org.apache.directory,net.sf.ehcache,org.apache.directory.api.ldap,javax.security.sasl,org.wildfly.security,org.wildfly.security.xml,org.wildfly.security.tls,org.wildfly.security.sasl
```

Comment:
as there are no meaningful changes, please remove any changes from this file to minimize patch size (smaller patch is better)